console/src/test/resources/Solidity/test3.sol

```diff
@@ -0,0 +1,40 @@
+pragma solidity ^0.4.25;
+
+// Modified Greeter contract. Based on example at https://www.ethereum.org/greeter.
+
+contract Mortal {
+    /* Define variable owner of the type address*/
+    address owner;
+
+    /* this function is executed at initialization and sets the owner of the contract */
+    constructor () public { owner = msg.sender; }
+
+    /* Function to recover the funds on the contract */
+    function kill() public { if (msg.sender == owner) selfdestruct(owner); }
+}
+
+contract Test3 is Mortal {
```

Comment:
ok - I figured out why you need this - you need to sort out the naming of these contracts.

Reply:
Renamed the contracts and deleted a few. The dummy contacts are there so I can test that the project importer can copy the files from a nested file structure.